index-import.cpp

```diff
@@ -172,6 +172,19 @@ static IndexUnitWriter remapUnit(const std::unique_ptr<IndexUnitReader> &reader,
   return writer;
 }
 
+static int cloneRecord(StringRef from, StringRef to) {
+  // Two record files of the same name are guaranteed to have the same
+  // contents, because the filename contains a hash of its contents. If the
+  // destination record file already exists, no action needs to be taken.
+  if (fs::exists(from)) {
```

Comment:
Won't `from` _always_ exist? I think this should be `to` no?

Reply:
thanks for reviewing and catching this! https://github.com/lyft/index-import/pull/20